compiler/rustc_middle/src/ty/instance.rs

```diff
@@ -365,6 +366,7 @@ impl<'tcx> Instance<'tcx> {
     }
 
     #[inline]
+    /// Returns the `def_id` of this instance.
```

Comment:
Nit: We generally do it the other way around, the do-comment before all the attributes.
```suggestion
    /// Returns the `def_id` of this instance.
    #[inline]
```